config/config.go

```diff
@@ -27,14 +28,18 @@ const (
 
 var defaultPaths = []string{rootVolume, rootUse, rootPolicy, rootPolicyArchive, rootSnapshots}
 
-// Request provides a request structure for communicating with the
-// apiserver.
-type Request struct {
-	Volume  string            `json:"volume"`
+// VolumeRequest provides a request structure for communicating volumes to the
+// apiserver or internally. it is the basic representation of a volume.
+type VolumeRequest struct {
```

Comment:
This is related to the config library (where database operations are performed) the structs in the docker library are just API exchange requests.